tests/drivers/gpio/gpio_api_1pin/boards/native_posix.overlay

```diff
@@ -7,3 +7,20 @@
 &gpio0 {
 	ngpios = <2>;
 };
+
+/ {
+	gpio-keys {
+		compatible = "zephyr,gpio-keys";
+
+		voldown_button: button_0 {
+			gpios = <&gpio0 0 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
```

Comment:
It might be smart to emphasize that we can have more than one button per device, as long as they have the same debounce constraint. I could imagine a common use case being e.g. a 3x3 array of buttons (device) all with a common debounce constraint, and then another gpio-keys device being a single button with a special debounce constraint.

Does that work?

Reply:
I added a `gpio-keys1` group with a different debounce interval below.  Is this what you are looking for?

Reply:
That works for me